docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -137,9 +139,33 @@ View Alluxio WebUI at `http://master_hostname:19999` and you can see the directo
 
 Alternatively, you can follow the [instructions](Running-Hive-with-Alluxio.html#create-new-tables-from-files-in-alluxio) to create the tables from existing files in Alluxio.
 
-Next, using a single query:
+Next, start your Hive metastore service. Hive metastore listens on port 9083 by default.
+
+```bash
+$ /<PATH_TO_HIVE>/bin/hive --service metastore
+```
+
+The following is an example of the Presto configuration `/<PATH_TO_PRESTO>/etc/catalog/hive.properties` :
+
+```properties
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://localhost:9083
+hive.config.resources=/<PATH_TO_HADOOP>/etc/hadoop/core-site.xml,/<PATH_TO_HADOOP>/etc/hadoop/hdfs-site.xml
 ```
-/home/path/presto/presto-cli-0.191-executable.jar --server masterIp:prestoPort --execute "use default;select * from u_user limit 10;" --user username --debug
+
+Start your Presto server. Presto server runs on port 8080 by default:
+
+```bash
+$ /<PATH_TO_PRESTO>/bin/launcher run
+```
+
+Download the `presto-cli-<YOUR_PRESTO_VERSION>-executable.jar` from [Presto CLI guidence](https://prestodb.io/docs/current/installation/cli.html) 
+and rename it to presto.
```

Comment:
why do you have to rename it to presto? Where do you have to save this file?

Reply:
Sometimes the ./bin/presto is directly installed, like brew install presto. The executable jar is already downloaded, renamed and made executable.

directly using an executable.jar make me feel confusing at the beginning until I saw the official docs.

Rename it to presto is to sync with Presto official docs and make new users like me who already have the renamed presto executable file inside the installation folder easier to figure out the presto file could be used.

changed to ./presto